layouts/partials/navbar.html

```diff
@@ -104,39 +104,30 @@
     
           <span class="navbar-item is-size-3 has-text-weight-light is-uppercase">
             SPIRE
           </span>
           
-          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Understand" "section" $spireUnderstand "pageUrl" $pageUrl "version" $version ) }}
-          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Installing" "section" $spireInstalling "pageUrl" $pageUrl "version" $version ) }}
-          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Using" "section" $spireUsing "pageUrl" $pageUrl "version" $version ) }}
-          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Developing" "section" $spireDeveloping "pageUrl" $pageUrl "version" $version ) }}
+          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "About" "section" $spireAbout "pageUrl" $pageUrl "version" $version ) }}
+          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Try"  "section" $spireTry "pageUrl" $pageUrl "version" $version ) }}
+          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Plan" "section" $spirePlanning "pageUrl" $pageUrl "version" $version ) }}
+          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Deploy" "section" $spireDeploying "pageUrl" $pageUrl "version" $version ) }}
+          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Maintain" "section" $spireMaintenance "pageUrl" $pageUrl "version" $version ) }}
           
           <span class="navbar-item is-size-3 has-text-weight-light is-uppercase">
-            SPIRE Integrations
+            Specific Integrations
           </span>
           
-          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Overview" "section" $spireInt "pageUrl" $pageUrl "version" $version ) }}
-          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "OIDC Federation" "section" $spireOIDC "pageUrl" $pageUrl "version" $version ) }}
-          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "SPIRE and Envoy" "section" $spireEnvoy "pageUrl" $pageUrl "version" $version ) }}
-          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "SPIFFE Libraries" "section" $spireLibraries "pageUrl" $pageUrl "version" $version ) }}
-          
-          <span class="navbar-item is-size-3 has-text-weight-light is-uppercase">
-            Reference
-          </span>
-          
-          <a class="navbar-item" href="https://docs.google.com/document/d/1RZnBfj8I5xs8Yi_BPEKBRp0K3UnIJYTDg_31rfTt4j8/" target="_blank">
-            Design Document: SPIFFE Reference Implementation (SRI)
-          </a>
-    
+          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "SPIRE Architecture" "section" $spireArchitecture "pageUrl" $pageUrl "version" $version ) }}
```

Comment:
Of the three buckets of integrations - this one is arguably the most advanced, and should go last. Consider also aligning the language of this heading to the "Build, bridge and extend" language on the homepage.

Reply:
I'm imagining this "SPIRE Architecture" as a separate integration category describing SPIRE architectural options by itself. To me, the "Build, bridge and extend" category seems to apply to service meshes. We don't currently have an integration example for service meshes. My [outline](https://docs.google.com/document/d/1ahjPB0r8wFwis4AwuDsL4QLtqOpxRtBoCqDtUKWjnm0/edit?usp=sharing) proposes adding one for Istio eventually but currently there's no bucket that matches that category on the home page. But I'm open to your take on this.

Reply:
Fair enough. Even so, I'd argue that Nested And Federated SPIRE are advanced use cases, whereas "securing microservices" and "keyless authentication" are entry level motivating use cases, and it would be better to put them first.